client/src/Picks.tsx

```diff
@@ -1004,6 +1005,32 @@ function ForceBeatmapsetUpdate({ nomination }: { nomination: INomination }) {
   );
 }
 
+function visualizeDescriptionDifferences(oldDescription: string, newDescription: string): string {
+  if (oldDescription === newDescription) {
+    return newDescription;
+  }
+
+  let display = '';
+  const diffMatch = new diff_match_patch();
+  const differences = diffMatch.diff_main(oldDescription, newDescription);
+
+  // cleanup
+  diffMatch.diff_cleanupSemantic(differences);
+
+  // compare
+  for (const difference of differences) {
+    if (difference[0] === 0) {
+      display += difference[1];
+    } else if (difference[0] === -1) {
+      display += `<span style="color: #854242; background-color: #AE8686;">${difference[1]}</span>`;
+    } else if (difference[0] === 1) {
+      display += `<span style="color: #698542; background-color: #95AE86;">${difference[1]}</span>`;
```

Comment:
should probably use & style `<del>` and `<ins>` instead.

Reply:
was going to, left this temporarily just for displaying